scripts/variantstore/variant-annotations-table/GvsCreateVATFilesFromBigQuery.wdl

```diff
@@ -225,7 +225,7 @@ task MergeVatTSVs {
         String cloud_sdk_slim_docker
     }
 
-    File monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
+    File monitoring_script = "gs://gvs_quickstart_storage/vs_1550_cromwell_monitoring_script.sh"
```

Comment:
I presume you are going to name this something differently before this is merged?

Reply:
Yeah I'd like to discuss in post tomorrow how this thing should be named going forward.